docs/documentation/pages_en/quickstart.md

```diff
@@ -265,15 +267,22 @@ Or use one of the following instructions to set up the local Kubernetes cluster
     
    **IMPORTANT** Param `--insecure-registry` allows usage of Container Registry without TLS. TLS in our case dropped for simplicity.
 
-3. Install NGINX Ingress Controller:
+3. If you haven't installed `kubectl` yet, install it by running:
+
+   ```
+   minikube kubectl -- get po -A
+   alias kubectl="minikube kubectl --"
```

Comment:
Shouldn't we add this alias into our `.bash_aliases` or `.bashrc`?

Reply:
Yes, that's a good idea!